docker-compose.yaml

```diff
@@ -0,0 +1,53 @@
+version: "3"
+
+services:
+  app:
+    build:
+      context: ../app/.
+    depends_on:
+      - api
+    container_name: app
+    restart: unless-stopped
+    environment:
+      NODE_ENV: "dev"
+    ports:
+      - "3000:3000"
+    volumes:
+      - ../app/:/usr/src/app
+    command: yarn dev
+
+  api:
+    build:
+      context: ../api/.
+    depends_on:
+      - database
+    container_name: api
+    restart: unless-stopped
+    env_file: ../api/.env
+    environment:
+      DATABASE_HOST: "database"
+      DATABASE_PORT: 3306
+      DATABASE_NAME: "cantodarua"
+      DATABASE_USERNAME: root
+      DATABASE_PASSWORD: "secret"
+      DATABASE_SSL: "false"
+      NODE_ENV: "develop"
+    ports:
+      - "1337:1337"
+    volumes:
+      - ../api/:/home/node/app
+    command: npm run develop
+
+  database:
+    image: mysql:5.6.35
```

Comment:
Why are we using mysql instead of postgresql? Any advantage?

Reply:
Just simplicity, but I'm in favour of postgres.

Reply:
also would like a sync